jupyterlab/upgrade_extension.py

```diff
@@ -116,7 +116,7 @@ def update_extension(  # noqa
 
     # Build up the template answers and run the template engine
     author = data.get("author", "<author_name>")
-    author_email = "<author_email>"
+    author_email = "author_email@jupyter.com"
```

Comment:
Could we use empty instead of a dummy placeholder? That case is also available in the extension template.

```suggestion
    author_email = ""
```